packages/shared/src/lib/image.ts

```diff
@@ -1,4 +1,14 @@
 export const cloudinary = {
+  feed: {
+    bg: {
+      laptop:
+        'https://daily-now-res.cloudinary.com/image/upload/v1665224467/public/Desktop-ask-bg.png',
```

Comment:
All of these seem to be wrong, missing the f_auto can you ensure with design please?
Or I can help get them if not

Reply:
They shared one with it, but it wasn't working, not sure if the files are not yet public. Should be found here: https://dailydotdev.slack.com/archives/C05EM7ZFQMQ/p1692271488033119?thread_ts=1692265064.294219&cid=C05EM7ZFQMQ